packages/components/src/components/checkbox/checkbox.tsx

```diff
@@ -1,48 +1,53 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import Icon from '../icon';
 import Text from '../text';
 
-const Checkbox = React.forwardRef(
+type TCheckBoxProps = Omit<React.HTMLProps<HTMLInputElement>, 'value'> & {
+    className?: string;
+    classNameLabel?: string;
+    defaultChecked?: boolean;
+    disabled?: boolean;
+    greyDisabled?: boolean;
```

Comment:
```suggestion
    disabled: boolean;
    greyDisabled: boolean;
```

Reply:
@farzin-fs **`?:`** is required for both of these props because based on this component's usage across the app, they are **non-required** props.

Reply:
@maryia-binary Yes you are correct, My bad 🙇🏻 

@iman-fs Please ignore these comments 🙏🏻